tests/testlib/s2n_test_server_client.c

```diff
@@ -78,7 +78,7 @@ S2N_RESULT s2n_negotiate_test_server_and_client_with_early_data(struct s2n_conne
     do {
         if (!client_early_done) {
             bool success = s2n_send_early_data(client_conn,
-                                   early_data_to_send->data + total_data_sent,
+                                   (early_data_to_send->data == NULL) ? NULL : early_data_to_send->data + total_data_sent,
                                    early_data_to_send->size - total_data_sent,
```

Comment:
I don't think this is a good use of a ternary operator. This will likely be more readable with a named variable declared outside of the s2n_send_early_data function call.